scapy/arch/windows/__init__.py

```diff
@@ -72,11 +72,39 @@ def _encapsulate_admin(cmd):
     # rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+def _hack_windows_subprocess():
+    """HACK: python 2.7 file descriptors.
+
+    This magic hack fixes https://bugs.python.org/issue19575
+    by adding HANDLE_FLAG_INHERIT to all already opened file descriptors.
+    """
+    # See https://github.com/secdev/scapy/issues/1136
+    import stat
+    from ctypes import windll, wintypes
+    from msvcrt import get_osfhandle
+
+    HANDLE_FLAG_INHERIT = 0x00000001
```

Comment:
It's available already in win32api, I think (possibly elsewhere).
Could it be reused?

Reply:
win32api is not a standard and therefore should not be used in scapy.
Scapy is a strict autonomous package (dependent only on standard library) so correct
way is using standard ctypes instead of external and very heavy pywin 32 package.